### DISCUSSION TITLE
I can't activate oversampling on Fabfilter plugins

### DISCUSSION THREAD

**Discussion by kastorjunior** [apohl79/audiogridder#363] [Help]
Hi ;)
Everything works great for me On Reaper and Studio One but I can't activate oversampling on ProMB or Pro-c from fabfilter on my production PC.
I can do it on the serveur. I think it's a GUI bug. thx. (last release 1.1.0)
https://www.casimages.com/i/210207083130862963.gif.html

**Comment by apohl79** [comment]
Is this a double click or single click action (on the server side where it works) ?

**Reply by kastorjunior** [reply]
Hi
 Simple left click to choose the amount of oversampling. Here a gif of my PC server
https://www.casimages.com/i/21020907344045741.gif.html

**Comment by apohl79** [comment]
I see and the left click through the AG plugin simply does nothing? If you extend the capturing area (click + at the top right) and try it again, do you see the popup than? it looks a bit like it might just be outside the capturing area.

**Reply by kastorjunior** [reply]
Thank you. I didn't see this + feature. Now I can see all my parameters in those plugins. Great work.!

**Reply by kastorjunior** [reply]
Another question.
Do you allow the creation of tutorials on a youtube channel about your work? With links to your site, of course, and donation links as well. :)

**Reply by apohl79** [reply]
> Another question.
> 
> Do you allow the creation of tutorials on a youtube channel about your work? With links to your site, of course, and donation links as well. :)

absolutely! if you share them, i can also link them from the homepage!

**Comment by CaBuffalo** [comment]
I don't see any problems? :):)

On Tue, Feb 9, 2021 at 12:46 PM kastorjunior <notifications@github.com>
wrote:

> Another question.
> Do you allow the creation of tutorials on a youtube channel about your
> work? With links to your site, of course, and donation links as well. :)
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/363#discussioncomment-354512>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6DIXBFDUTVPZ7XTJPTS6GNLVANCNFSM4XHYSPCQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.
